listenbrainz/webserver/templates/index/follow.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html' %}
+
+{% block title %}Follow users {{ follow_list|join(', ') }} - ListenBrainz{% endblock %}
+
+{% block content %}
+
+    <div id="react-listens">
+    </div>
+
+{% endblock %}
+
+{% block scripts %}
+  {{ super() }}
+  <script defer src="https://use.fontawesome.com/releases/v5.0.8/js/all.js" integrity="sha384-SlE991lGASHoBfWbelyBPLsUlwY1GwNDJo3jSJO04KZ33K2bwfV9YBauFfnzvynJ" crossorigin="anonymous"></script>
```

Comment:
is this code reused in another template? I'm not sure if it is, but we should try to not duplicate it if it is being duplicated.

Reply:
Yes, here:
https://github.com/metabrainz/listenbrainz-server/blob/5c3d6fdab340180111e8a16ec5d62c5484a0f3d9/listenbrainz/webserver/templates/user/profile.html#L19

What's the best way to refactor?
Create and extend a react template?

Reply:
OK, I refactored with a react base template